device/peer.go

```diff
@@ -267,10 +259,8 @@ func (peer *Peer) Stop() {
 }
 
 func (peer *Peer) SetEndpointFromPacket(endpoint conn.Endpoint) {
-	if peer.disableRoaming {
+	if peer.disableRoaming.Load() {
 		return
 	}
-	peer.Lock()
-	peer.endpoint = endpoint
-	peer.Unlock()
+	peer.endpoint.Store(endpoint)
```

Comment:
is this hot?

were you going to do the thing where you say a conn.Endpoint must be comparable (it usually is, if it's backed by a pointer (to a struct)) and then do a Load first and bail out if it's unchanged, rather than jumping to a store?